### PR TITLE
added info for batu and myself!

### PR DESCRIPTION
**Pull request by drboyd**
Please let me know if this doesn't work! Thanks!

### CODE REVIEW

File: book/team.yaml [hunk 1]
```diff
@@ -203,4 +203,40 @@ people:
         link: https://github.com/jomey
     user_groups:
       - Technology Specialist
+  # =============
+  - title: Dylan Boyd
+  avatar: https://avatars.githubusercontent.com/u/28358098?v=4
+  role: Tutorial Lead
+  organizations:
```
Comment (scottyhq) [inline book/team.yaml]: thanks @db1950! indentation levels matter for formatting in this file, please match the indentation against earlier entries

Comment (drboyd) [reply book/team.yaml]: Hi @scottyhq. Looks like it's fixed now. I apologize!

**Comment by scottyhq** [comment]
@db1950 you just need to fix the merge conflicts since others have also modified the same file (see thread on snowex slack workspace)

**Comment by drboyd** [comment]
Sorry for the delay, Scott! I did some reading on top of your how-to. Thanks for teaching us how to fish.

**Review by scottyhq — APPROVED** [review]
thanks!

**Pull request merged**